### PR TITLE
WPF-910709 : Update the ToolBarStateChangedEvent description in ToolBarAdv's  UG documentation in WPF.

### PR DESCRIPTION
**Pull request by nithisam22**
### Description:

Update the ToolBarStateChangedEvent description in ToolBarAdv's UG documentation in WPF.

### CODE REVIEW

File: wpf/ToolBar/ToolBarAdv-state.md [hunk 1]
```diff
@@ -189,3 +189,74 @@ toolBarManager.CanDockAtTop = false;
 {% endhighlight %}
 
 {% endtabs%}
+
+## Implementation of toolbar state changed event to detect when the state of ToolBarAdv changes.
+
```
Comment (Manivannan-E) [inline wpf/ToolBar/ToolBarAdv-state.md:194]: @nithisam22 
## StateChanged event

Change the content and code snippet as we discussed.

File: wpf/ToolBar/ToolBarAdv-state.md [hunk 2]
```diff
@@ -174,18 +119,97 @@ Following code restricts docking at the top:
...
+## ToolBar State Changed Event
 
-ToolBarManager toolBarManager = new ToolBarManager(); 
+ToolBarStateChanged event is triggered when the state of a ToolBarAdv changes, such as when it is hidden, floated, or docked. This event passes an instance of ToolBarStateChangedEventArgs, which contains information about the state change, including the old state and new state of the toolbar.
 
-toolBarManager.CanDockAtTop = false;
+NewState: Gets the current state of the toolBarAdv control.
 
+OldState: Gets the previous state of the toolBarAdv control.
```
Comment (Manivannan-E) [inline wpf/ToolBar/ToolBarAdv-state.md:139]: @nithisam22 

The ToolBarStateChanged event is triggered when the state of a ToolBarAdv changes, such as when it is hidden, floated, or docked. This event passes an instance of ToolBarStateChangedEventArgs, which contains information about the state change, including the old state and the new state of the toolbar.

NewState: Gets the current state of the ToolBarAdv control.
OldState: Gets the previous state of the ToolBarAdv control.

Comment (nithisam22) [reply wpf/ToolBar/ToolBarAdv-state.md]: @Manivannan-E sir, changes have been made

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

**Pull request merged**